src/components/forms/PhoneInput.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from "react"
+import parsePhoneNumber from "libphonenumber-js"
```

Comment:
I think the linter here wants you to use 
```suggestion
import { parsePhoneNumber } from "libphonenumber-js"
```